website/mails/mails.py

```diff
@@ -372,3 +372,8 @@ def get_english_article(word):
     'send_data_share_preprint_error_desk',
     subject='Share Error'
 )
+
+REVIEWS_SUBMISSION_CONFIRMATION = lambda provider_name: Mail(
```

Comment:
The `lambda` shouldn't be necessary. The subject can be a string template that receives the same context as the email body.

For example

```python
FILE_OPERATION_SUCCESS = Mail(
    'file_operation_success',
    subject='Your ${action} has finished',
)
```